tests/test_enum_total_ordering.py

```diff
@@ -49,7 +49,11 @@ def test_total_ordering_w_other_enum_type():
 
     for item in enums_test.OtherEnum:
         assert not to_compare == item
-        assert to_compare.SOME_VALUE != item
+        assert type(to_compare).SOME_VALUE != item
```

Comment:
`to_compare` is already `OneEnum.SOME_VALUE` -- so what is the purpose of the indirect reference through itself?  Can this part of the test just be dropped?